packages/client/src/clients/objects/index.ts

```diff
@@ -0,0 +1,63 @@
+import { ApiResponse } from "../../api";
+import { ChannelData } from "../../interfaces";
+import Knock from "../../knock";
+
+type GetChannelDataInput = {
+  objectId: string;
+  collection: string;
+  channelId: string;
+};
+
+type SetChannelDataInput = {
+  objectId: string;
+  collection: string;
+  channelId: string;
+  data: any;
+  userId: string;
+};
+
+class ObjectClient {
```

Comment:
Added this client so that we could move these generic calls out of the slack client, which previously had the `setConnectedChannels` function that essentially was just setting channel data.